config/settings/base.py

```diff
@@ -20,10 +20,10 @@
     'corsheaders',
     'rest_framework',
 
-    'v1.meta.apps.MetaConfig',
-    'v1.opening.apps.OpeningConfig',
-    'v1.task.apps.TaskConfig',
-    'v1.team.apps.TeamConfig'
+    'v1.teams.apps.TeamsConfig',
```

Comment:
if i'm not mistaking, the ordering here should be like
contributors
openings
tasks 
teams